use crate::shared::errors::Error;


pub async fn delete_column() -> Result<String, Error> {

    Ok(format!("Success"))
}